samples/simd/simd_test.kt

```diff
@@ -1,87 +0,0 @@
-import kotlinx.cinterop.*
```

Comment:
I guess we still need some tests.
Considering that `NativeVector` becomes first-class type with "intrinsified" implementation, I guess we need 
both 
1. Interop-less tests for `NativeVector` type itself 
2. Interop-specific tests for passing vectors between Kotlin and C.